ECHO-COVID19.ipynb

```diff
@@ -281,45 +353,21 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "data['HPV_DAYZERO_DATE'] = pd.to_datetime(data['HPV_DAYZERO_DATE'], format=\"%m-%d-%Y\") #format\n",
-    "cps = data\n",
-    "\n",
-    "cps = cps[(cps['POLLUTANT_DESCS'].astype(str).str.contains('Particulate')) | (cps['POLLUTANT_DESCS'].astype(str).str.contains('Sulfur'))]\n",
-    "cps_map = cps # for mapping later...\n",
-    "\n",
-    "cps = cps.groupby(['HPV_DAYZERO_DATE'])['POLLUTANT_DESCS'].count() \n",
-    "# = total number of violations. May want to summarize by facility instead. \n",
-    "\n",
-    "cps = cps.resample('M').sum() #resample to a monthly basis  \n",
-    "cps = cps.loc[(cps.index.month == int(mnth))] # Filter back to just Aprils\n",
-    "cps = pd.DataFrame(cps)\n",
-    "cps = cps.rename(columns={'POLLUTANT_DESCS': \"Number of violations\"})\n",
-    "cps.index = cps.index.strftime('%Y-%m') # makes the x axis (date) prettier\n",
-    "\n",
-    "ax = cps.plot(kind='bar', title = \"# of Violations Related to Particulate Matter and/or Sulfur Dioxide in %s Across Years\" %(mnth_name), figsize=(20, 10), fontsize=16)\n",
-    "ax\n",
-    "\n",
-    "#label trendline\n",
-    "trend=cps['Number of violations'].mean()\n",
-    "ax.axhline(y=trend, color='#e56d13', linestyle='--', label = 'Average # of violations in %s across ECHO history' %(mnth_name))\n",
-    "\n",
-    "#label past 4 Aprils trend (2017, 2018, 2019, 2020)\n",
-    "trend_month=pd.concat([cps.loc[\"2017-\"+mnth],cps.loc[\"2018-\"+mnth],cps.loc[\"2019-\"+mnth]])\n",
-    "trend_month=trend_month['Number of violations'].mean()\n",
-    "ax.axhline(y=trend_month, color='#d43a69', linestyle='--', label = 'Average # of violations in %s 2017-2019' %(mnth_name))\n",
+    "sql = \"select * from `ICIS-AIR_VIOLATION_HISTORY` where HPV_DAYZERO_DATE like '\"+mnth+\"-%'\"\n",
+    "url='http://apps.tlt.stonybrook.edu/echoepa/?query='\n",
+    "data_location=url+urllib.parse.quote(sql)\n",
     "\n",
-    "# Label plot\n",
-    "ax.legend()\n",
-    "ax.set_xlabel(\"Month and year of violations\")\n",
-    "ax.set_ylabel(\"Number of violations in %s across years\" %(mnth_name))"
+    "air_data = pd.read_csv(data_location,encoding='iso-8859-1',header = 0)\n",
+    "air_data.set_index(\"pgm_sys_id\", inplace=True)\n",
+    "air_data"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### *Where* are these facilities that exceeded their PM and SO2 permits last month?\n",
-    "Even if, on the whole, there are fewer exceedances, the places that are emitting more pollutants are important to track. Their neighbors are suffering more."
+    "#### Chart emissions violations\n",
+    "The height of each bar indicates how many emissions violations there have been. The red line shows the average number of emissions violations for all previous years, and the yellow line indicates the average for the past three years."
```

Comment:
Colors changed, it's now orange and pink where it used to be (respectively) red and yellow